zerver/openapi/zulip.yaml

```diff
@@ -1916,12 +1917,93 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/JsonSuccess'
+                allOf:
+                - $ref: '#/components/schemas/JsonSuccess'
+                - example:
+                      {
+                          "msg": "",
+                          "result": "success"
+                      }
+        '400':
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/JsonError'
+                - example:
+                    {
+                        "code": "BAD_REQUEST",
+                        "msg": "Invalid stream id",
+                        "result": "error"
+                    }
+    patch:
+      description: Update the stream with the given ID.
+      parameters:
+      - name: stream_id
+        in: path
+        description: The ID of the stream to be updated.
+        schema:
+          type: integer
+        example: 42
+        required: true
+      - name: description
+        in: query
+        description: The new description that is to be updated.
```

Comment:
I think "The new description that is to be updated" is awkward phrasing.  I think 

"The new description for the stream" 

is a lot clearer.  (Same feedback for the other N)

Reply:
Updated for this as well others.